docs/indexing.md

```diff
@@ -50,6 +50,16 @@ You can select a collection and use the "query" menu option to check how many do
 
 ## Creating a Full Index
 
+### Check solr cloud health
```

Comment:
@sandbergja can we now just link to this section in the bottom?

Reply:
@sandbergja it looks great. Can we move Check solr cloud health before the swap section https://github.com/pulibrary/bibdata/blob/d3d64ec800cb3b814a77b952dfe957db6cf5025e/docs/indexing.md#swap-in-the-new-index